Package/dotnettool/IronPython.Console.csproj

```diff
@@ -0,0 +1,56 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project Sdk="Microsoft.Build.NoTargets/3.3.0">
+
+  <PropertyGroup>
+    <TargetFramework>net6.0</TargetFramework>
+    <OutputType>Exe</OutputType>
+    <AssemblyName>ipy</AssemblyName>
+    <UseAppHost>false</UseAppHost>
+    <IncludeSymbols>false</IncludeSymbols>
+    <IncludeBuildOutput>false</IncludeBuildOutput>
+    <PackageOutputPath>$(PackageDir)</PackageOutputPath>
+    <NoWarn>$(NoWarn);NU5110;NU5111</NoWarn>
+
+    <PackAsTool>true</PackAsTool>
+    <ToolCommandName>ipy</ToolCommandName>
+    <PackageType>DotnetTool</PackageType>
+    <RestoreProjectStyle>DotnetToolReference</RestoreProjectStyle>
+  </PropertyGroup>
+
+  <PropertyGroup>
+    <!-- nuspec properties -->
+    <PackageId>IronPython.Console</PackageId>
+    <Title>IronPython Command-line Interpreter</Title>
+    <Version>$(PackageVersion)</Version>
+    <Authors>IronPython Contributors, Microsoft</Authors>
+    <PackageProjectUrl>https://ironpython.net</PackageProjectUrl>
+    <RepositoryType>git</RepositoryType>
+    <RepositoryUrl>https://github.com/IronLanguages/ironpython3.git</RepositoryUrl>
+    <PackageLicenseExpression>Apache-2.0</PackageLicenseExpression>
+    <PackageRequireLicenseAcceptance>false</PackageRequireLicenseAcceptance>
+    <Description>IronPython is an open-source implementation of the Python programming language that is tightly integrated with the .NET Framework. IronPython can use the .NET Framework and Python libraries, and other .NET languages can use Python code just as easily.
+
+This package contains a standalone Python interpreter, invokable from the command line as ipy. It also includes the Python Standard Library released by the Python project, but slightly modified to work better with IronPython and .NET.</Description>
+    <PackageTags>ironpython python dynamic dlr</PackageTags>
+    <PackageReadmeFile>README.md</PackageReadmeFile>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="DynamicLanguageRuntime" Version="1.3.2" />
```

Comment:
Wondering if we should grab the DLR from staging, the advantage being that we don't need a release in order to do a build with DLR updates.

Reply:
I'm going to park this option for now; depending on the outcome of the versioning discussion, it might become a useful option.